markdown/blog/2025-singapore-conf-summary.md

```diff
@@ -0,0 +1,175 @@
+  className="text-center"
+/>
+
+Next was [Tamimi Ahmad](https://www.linkedin.com/in/atamimi1/), Senior Developer Advocate at [Solace](https://www.linkedin.com/company/solacedotcom/) presenting "The Untapped Potentials of AsyncAPI Studio"
+
+Tamimi showcased AsyncAPI Studio’s newest capabilities—making async design more accessible, visual, and powerful than ever. His live walkthrough had folks taking notes and asking how they could integrate the Studio into their team workflows.
+
+<Figure
+  src="/img/posts/marketing-images/singapore-conf/tamimi.webp"
+  caption="Tamimi presenting the Untapped Potentials of AsyncAPI Studio."
+  className="text-center"
+/>
+
+It was followed by [Hugo Guerrero](https://www.linkedin.com/in/hugoguerrero/), Head of Technical Marketing at [Kong](https://www.linkedin.com/company/konghq/) speaking on "Enhancing Kafka Security and Control with Event Gateways"
+
+Hugo brought his deep expertise in Kafka to the table, explaining how event gateways provide much-needed visibility, governance, and access control in Kafka-based environments. Security in EDA isn’t optional—and this talk proved why.
+
+<Figure
+  src="/img/posts/marketing-images/singapore-conf/hugo.webp"
+  caption="Hugo presenting Enhancing Kafka Security and Control with Event Gateways."
+  className="text-center"
+/>
+
+We wrapped up the track with [Ankit Kumar](https://www.linkedin.com/in/ankit28/), Head of Developer & Customer Success at [Aklivity](https://www.linkedin.com/company/aklivity/) speaking on "Streamlining Event-Driven API Design"
+
+Ankit gave us an exciting dive into AsyncAPI v3—our next major milestone. He shared what’s changing, why it matters, and how v3 brings clarity and flexibility to designing and managing complex event-driven APIs.
+
+<Figure
+  src="/img/posts/marketing-images/singapore-conf/ankit.webp"
+  caption="Ankit presenting Streamlining Event-Driven API Design."
+  className="text-center"
+/>
+
+We had:
+
+- Real-world examples of AsyncAPI in action
+
+- Live demos using AsyncAPI Studio and the CLI
+
+- The tooling and ecosystem that surrounds the spec
+
+But more than the slides and demos, it was the questions that told the story:
+
+- "Tell me more about AsyncAPI"
+- “How do I introduce AsyncAPI into our hybrid systems?”
+- “Can I use this with Kafka and MQTT?”
+- “How do we contribute to the spec or tooling?”
+- "How do other AsyncAPI adopters use AsyncAPI?"
+
+We heard curiosity, but also readiness—this wasn’t just a learning moment; for many, it was a turning point.
+
+<Figure
+  src="/img/posts/marketing-images/singapore-conf/singapore-conference.webp"
+  caption="AsyncAPI Track in motion."
+  className="text-center"
+/>
+
+By the end of the conference, we had:
+
+- Welcomed over 20 new faces to our Slack and GitHub communities, though we also noticed some hesitation to join the Slack community.
+
+- Sparked contributor interest in tooling, docs, and protocol support
+
+## 💚 A Community-First Future
```

Comment:
We have always been community first, You might want to use a different term or reconsider this paragraph